sample/kotlin/src/main/kotlin/com/datadog/android/sample/sessionreplay/ImageComponentsFragment.kt

```diff
@@ -0,0 +1,114 @@
+    private lateinit var viewModel: ImageComponentsViewModel
+    private lateinit var textViewRemote: TextView
+    private lateinit var buttonRemote: Button
+    private lateinit var viewRemote: View
+    private lateinit var imageViewRemote: ImageView
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        val rootView = inflater.inflate(R.layout.fragment_image_components, container, false)
+        textViewRemote = rootView.findViewById(R.id.textViewRemote)
+        buttonRemote = rootView.findViewById(R.id.buttonRemote)
+        viewRemote = rootView.findViewById(R.id.viewRemote)
+        imageViewRemote = rootView.findViewById(R.id.imageView_remote)
+        return rootView
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        viewModel = ViewModelProvider(this).get(ImageComponentsViewModel::class.java)
+        loadRemoteViews()
+    }
+
+    // internal region
+
+    private fun loadRemoteViews() {
+        loadView()
+        loadImageView()
+        loadButton()
+        loadTextView()
+    }
+
+    private fun loadView() {
+        viewModel.fetchRemoteImage(
+            largeImageUrl,
+            viewRemote,
+            object : ImageLoadedCallback {
+                override fun onImageLoaded(resource: Drawable) {
+                    viewRemote.background = resource
+                }
+            }
+        )
+    }
+
+    private fun loadImageView() {
+        viewModel.fetchRemoteImage(
+            largeImageUrl,
+            imageViewRemote,
+            object : ImageLoadedCallback {
+                override fun onImageLoaded(resource: Drawable) {
+                    imageViewRemote.setImageDrawable(resource)
+                }
+            }
+        )
+    }
+
+    private fun loadButton() {
+        viewModel.fetchRemoteImage(
+            smallImageUrl,
+            buttonRemote,
+            object : ImageLoadedCallback {
+                override fun onImageLoaded(resource: Drawable) {
+                    buttonRemote.setCompoundDrawablesWithIntrinsicBounds(resource, null, null, null)
+                }
+            }
+        )
+    }
+
+    private fun loadTextView() {
+        viewModel.fetchRemoteImage(
+            smallImageUrl,
+            textViewRemote,
+            object : ImageLoadedCallback {
+                override fun onImageLoaded(resource: Drawable) {
+                    textViewRemote.setCompoundDrawablesWithIntrinsicBounds(resource, null, null, null)
+                }
+            }
+        )
+    }
+
+    // endregion
+
+    companion object {
+        private const val smallImageUrl = "https://avatars.githubusercontent.com/u/365230?s=200&v=4"
+        private const val largeImageUrl = "https://imgix.datadoghq.com//img/about/presskit/Bits_Mascot.jpg"
```

Comment:
naming convention + extra `/`.

```suggestion
        private const val SMALL_IMAGE_URL = "https://avatars.githubusercontent.com/u/365230?s=200&v=4"
        private const val LARGE_IMAGE_URL = "https://imgix.datadoghq.com/img/about/presskit/Bits_Mascot.jpg"
```